policyengine_us/variables/gov/states/ct/tax/income/subtractions/ct_social_security_benefit_adjustment.py

```diff
@@ -0,0 +1,33 @@
+from policyengine_us.model_api import *
+
+
+class ct_social_security_benefit_adjustment(Variable):
+    value_type = float
+    entity = TaxUnit
+    unit = USD
+    label = "Connecticut social security benefit adjustment"
+    definition_period = YEAR
+    defined_for = StateCode.CT
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.ct.tax.income.subtractions.social_security
+        filing_status = tax_unit("filing_status", period)
+        is_eligible = tax_unit(
+            "ct_social_security_benefit_adjustment_eligible", period
+        )
+        ss_benefit = add(tax_unit, period, ["social_security"])
+        agi = add(tax_unit, period, ["adjusted_gross_income"])
```

Comment:
remove the eligibility code from this file

Reply:
I did the AGI comparison in the eligibility file, I thinks that all we need (?)